### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by nayakswadhin**
Deployed url - https://hackodisha2023-lyart.vercel.app/swadhin

### CODE REVIEW

**Comment by 8rxn** [comment]
@nayakswadhin Improve Responsiveness of the navbar. It falls apart on mobile devices

**Comment by nayakswadhin** [comment]
> @nayakswadhin Improve Responsiveness of the navbar. It falls apart on mobile devices

@8rxn Changes updated

**Comment by 8rxn** [comment]
@nayakswadhin Great work. One thing to tweak here is 

[Screencast from 05-07-23 11:48:30 PM IST.webm](https://github.com/Hackodisha/Hackodisha2023/assets/75237697/5187d875-de25-441a-a59c-b7dc83d52e02)

---

Look into the video and try animating the button, 
P.S. You can look into the code for navbar from last years website.

File: package.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-lottie": "^1.2.3",
+    "sharp": "^0.32.1",
```
Comment (duckling69) [inline package.json:22]: are we actually using this module ? if not please uninstall

File: src/app/swadhin/layout.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import Navbar from "./components/Navbar";
+
+function HomeLayout({ children }) {
+  return (
+    <>
+      <Navbar />
+      {children}
```
Comment (duckling69) [inline src/app/swadhin/layout.js:8]: why are we using children prop here?

**Review by duckling69 — CHANGES_REQUESTED** [review]
Change all the pngs to svgs and use proper file structure

File: package.json (hunk 1 above)
Comment (nayakswadhin) [reply package.json:22]: Sharp is for image optimization in production internally used by Next js

File: src/app/swadhin/layout.js (hunk 2 above)
Comment (nayakswadhin) [reply src/app/swadhin/layout.js:8]: This is a layout children is used to render the page here.

**Comment by nayakswadhin** [comment]
> Change all the pngs to svgs and use proper file structure

@duckling69 anything specific you want to highlight on the file structure.

**Comment by nayakswadhin** [comment]
> @nayakswadhin Great work. One thing to tweak here is
> 
>  Screencast.from.05-07-23.11.48.30.PM.IST.webm 
> Look into the video and try animating the button, P.S. You can look into the code for navbar from last years website.

@8rxn Added the required animation in the HamBurger icon and also added some animation in menu bar

**Comment by nayakswadhin** [comment]
> Change all the pngs to svgs and use proper file structure

Converted all the pngs to svgs

**Comment by 8rxn** [comment]
@nayakswadhin Great work, Looks good.
Although there's one glitch so you'll have to create a new Fork and PR. I'll elaborate in the meeting.

**Pull request merged**